getmoda.io.domain.json

```diff
@@ -0,0 +1,44 @@
+{
+    "providerId": "getmoda.io",
+    "providerName": "Moda",
+    "serviceId": "domain",
+    "serviceName": "Moda Domain",
+    "logoUrl": "https://userconfig.getmoda.io/moda-logo/moda-icon.svg",
+    "description": "This is a template to enable domain verification on Moda",
+    "variableDescription": "cname_host is the mail CNAME. cname_points is the value to which mail CNAME points to. dkim_points and dkim2_points are for the DKIM records ",
+    "syncBlock": true,
```

Comment:
Sure about it? What's the point of `syncPubKeyDomain` and `syncPubKeyDomain` when you intend to block sync flow?

Reply:
Will make this false